### ISSUE TITLE
Fatal error: Class 'PromisePay' not found

### ISSUE THREAD

**Issue by mkucej** [PromisePay/promisepay-php#37]
In our environment, using

``` PHP
use PromisePay
```

results in an error and a warning:

```
Fatal error: Class 'PromisePay' not found
Warning: The use statement with non-compound name 'PromisePay' has no effect
```

This is solved by using instead:

``` PHP
use PromisePay\PromisePay as PromisePay
```

**Comment by nasir00376** [comment]
An uncaught Exception was encountered

Type: Error

Message: Class 'PromisePay\PromisePay' not found

Filename: D:\xamp\htdocs\fashion\application\controllers\PromisePay_controller.php

Line Number: 13

Backtrace:

File: D:\xamp\htdocs\fashion\index.php
Line: 315
Function: require_once

**Comment by NinoSkopac** [comment]
Importing in PHP works like this: `use <namespace>\[... optional aditional namespaces ...]\ClassName`.

Therefore, 
`use PromisePay\PromisePay as PromisePay`
is equal to
`use PromisePay\PromisePay`

**Comment by NinoSkopac** [comment]
Fixed in #40
